src/RepeaterChildSupport.js

```diff
@@ -40,6 +40,14 @@ var RepeaterChildSupport = {
 	selected: false,
 
 	/**
+	* The CSS class applied to the child when selected
+	*
+	* @type {String}
+	* @default null
+	* @public
+	*/
```

Comment:
I pulled it because it was overwriting the value I added in onyx. Adding the `@name` in its place.